setup.py

```diff
@@ -28,6 +28,7 @@
         "dev": [
             "pytest",
             "wheel",
+            "pylint>=2.13.9",
```

Comment:
You might want to pin the version. In other projects I've had CI break when pylint releases a new update because they add more checks.

Reply:
Yes, you are completely right. I will do so.